ecosystem/indexer-grpc/indexer-grpc-table-info/src/backup_restore/fs_ops.rs

```diff
@@ -0,0 +1,257 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use flate2::{read::GzDecoder, write::GzEncoder, Compression};
+use std::{
```

Comment:
note: std operations can block the main thread like `std::thread::sleep` if used with tokio tasks. 

should be fine with std threads.

Reply:
yep totally aware of that, in the next pr, creating a standalone spawn blocking thread to separately handle file operation and gcs upload, so that it won't block other concurrent async threads from running.

Reply:
oh my point is, you can use tokio::fs/tokio::io if IO operations needed in async context. 

spawn_blocking is for CPU intensive operations.